src/jit/decomposelongs.cpp

```diff
@@ -946,6 +949,72 @@ GenTree* DecomposeLongs::DecomposeShift(LIR::Use& use)
 }
 
 //------------------------------------------------------------------------
+// DecomposeMul: Decompose GT_MUL
+//
```

Comment:
There should be a comment about how decompose will happen, in particular, that the GTF_MUL_64RSLT case is NOT handled here (and indicate where/how it is handled)